src/instances/classdef/index.js

```diff
@@ -0,0 +1,22 @@
+const Obj = require("../obj/index");
+
+const ClassDef = (() => {
+  const append = a => b => {
+    return {
+      mdefs: Obj.append(a.mdefs)(b.mdefs),
```

Comment:
Do we need to worry about overlap or weird stuff here?

Reply:
Yes, it would be bad if functions with the same name but different behavior overlapped.  I'm not sure what the best way to address this is, do we want to inspect the `mdefs` and warn/crash if there is an overlap?